app/res/layout/query_prompt_layout.xml

```diff
@@ -18,6 +18,15 @@
         android:gravity="center|left"
         android:layout_height="wrap_content" />
 
+    <ImageButton
+        android:id="@+id/prompt_hint_button"
+        android:layout_width="0dp"
```

Comment:
curious what happens if we set the width here to `wrap_content` and remove `layout_weight` from below ? I want to avoid using `layout_weight` for help icon since it may result in unexpected behaviours for different screen sizes. We are setting `layout_weight` on other views to `1` since we want them to take all the available space but this is not true for help view.